components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/jwks/JwksEndpoint.java

```diff
@@ -120,25 +121,79 @@ private String buildResponse(List<CertificateInfo> certInfoList)
         // If we read different algorithms from identity.xml then put them in a list.
         List<JWSAlgorithm> diffAlgorithms = findDifferentAlgorithms(accessTokenSignAlgorithm, config);
         // Create JWKS for different algorithms using new KeyID creation method.
+        getJWKArray(certInfoList, diffAlgorithms, jwksArray, OAuthConstants.SignatureAlgorithms.KID_HASHING_ALGORITHM);
```

Comment:
Just an improvement suggestion, a get method usually returns something.
If a get method doesn't return something its an indication, the method name is not right or does not reflect its true purpose. Can we improve the name?